src/test/tcp/CMakeLists.txt

```diff
@@ -1,9 +1,18 @@
+include_directories(${GLIB_INCLUDES})
+link_libraries(${GLIB_LIBRARIES})
```

Comment:
It doesn't look like you're actually using any glib functionality. If that's the case better to leave off the dependency for now.

Reply:
We need those dependencies with test_glib_helpers.h

Reply:
Oops, right, my mistake :)